lib/ExpensiMark.js

```diff
@@ -49,6 +49,39 @@ export default class ExpensiMark {
                 replacement: '$1<code>$2</code>$3',
             },
 
+            /**
+             * Apply the hereMention first because the string @here is still a valid mention for the userMention regex.
+             * This ensures that the hereMention is always considered first, even if it is followed by a valid userMention.
+            */
+            {
+                name: 'hereMentions',
+                regex: /((&#x60;|<code>|<pre>)\s*?)?[`.a-zA-Z]?@here\b/gm,
```

Comment:
I looked over the regex, but my understanding is not 100% as I'm still newbie with regrex. as long as regex is doing what comments are staying, then this PR LGTM!

just a question to deepen my regex knowledge: is the capturing group included in the `match`?

Reply:
thank you for your detailed explanation!